src/components/Footer/index.js

```diff
@@ -40,16 +128,52 @@ const Footer = () => (
           <div className='footer-secondary__data'>
             <ul className='footer-secondary__list'>
               <li className='footer-secondary__list-item'>
-                <a className='footer-secondary__link' href='https://docs.rockarch.org/argo-docs/'>Collections data API</a>
+                <a className='footer-secondary__link' href={
+                  t({
+                    comment: 'Collections data API link',
+                    message: 'https://docs.rockarch.org/argo-docs/'
+                  })
+                }>
+                  <Trans comment='Collections data API message'>
+                    Collections data API
+                  </Trans>
+                </a>
               </li>
               <li className='footer-secondary__list-item'>
-                <a className='footer-secondary__link' href='https://github.com/RockefellerArchiveCenter/data/'>Bulk data download</a>
+                <a className='footer-secondary__link' href={
+                  t({
+                    comment: 'Bulk Data Download link',
+                    message: 'https://github.com/RockefellerArchiveCenter/data/'
+                  })
+                }>
+                  <Trans comment='Bulk Data Download message'>
+                    Bulk data download
+                  </Trans>
+                </a>
               </li>
               <li className='footer-secondary__list-item'>
-                <a className='footer-secondary__link' href='https://docs.rockarch.org/archival-description-license/'>Licensing for descriptive metadata</a>
+                <a className='footer-secondary__link' href={
+                  t({
+                    comment: 'Licensing link',
+                    message: 'https://docs.rockarch.org/archival-description-license/'
+                  })
+                }>
+                  <Trans comment='Licensing message'>
+                    Licensing for descriptive metadata
+                  </Trans>
+                </a>
               </li>
               <li className='footer-secondary__list-item'>
-                <a className='footer-secondary__link' href='https://docs.rockarch.org/takedown-policy/'>Take-down policy</a>
+                <a className='footer-secondary__link' href={
+                  t({
+                    comment: 'Take-down Policy link',
+                    message: 'https://docs.rockarch.org/takedown-policy/'
+                  })
+                }>
+                  <Trans comment='Take-down Policy message'>
+                    Take-down policy
+                  </Trans>
+                </a>
```

Comment:
If we're going to go the route of creating a PrimaryLink component, I think we could potentially do the same here for SecondaryLink.